west.yml

```diff
@@ -113,7 +113,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 0fb04fb543c564430325b1df92c973a013632059
+      revision: pull/783/head
```

Comment:
We don't need this; we will need to merge this and rebase spin CI again on the `nrfxlib` PR instead.
That is because the compliance script checks out `sdk-nrf/main`